main/src/main/scala/sbt/EvaluateTask.scala

```diff
@@ -162,19 +162,23 @@ object EvaluateTask {
   import Keys.state
 
   lazy private val sharedProgress = new TaskTimings(reportOnShutdown = true)
-
-  // sbt-pgp calls this
-  private[sbt] def defaultProgress(): ExecuteProgress[Task] = ExecuteProgress.empty[Task]
-  private[sbt] def defaultProgress(currentRef: ProjectRef): ExecuteProgress[Task] =
+  def taskTimingProgress: Option[ExecuteProgress[Task]] =
     if (java.lang.Boolean.getBoolean("sbt.task.timings")) {
```

Comment:
Not directly related to this PR, but it seems like a bummer that this is a system property since system properties aren't really discoverable. I personally would have saved myself some time if I'd known about it.